scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -1,21 +1,37 @@
 #!/usr/bin/env -S dotnet fsi
 
 open System
+open System.IO
 open System.Net.Http
 open System.Net.Http.Headers
 
+#r "nuget: FSharp.Data, Version=5.0.2"
+open FSharp.Data
+
 #r "nuget: Fsdk, Version=0.6.0--date20230214-0422.git-1ea6f62"
 open Fsdk
 
+let githubEventPath = Environment.GetEnvironmentVariable "GITHUB_EVENT_PATH"
 
-let gitRepo = Environment.GetEnvironmentVariable "GITHUB_REPOSITORY"
-
-if String.IsNullOrEmpty gitRepo then
+if String.IsNullOrEmpty githubEventPath then
     Console.Error.WriteLine
         "This script is meant to be used only within a GitHubCI pipeline"
 
     Environment.Exit 2
 
+[<Literal>]
+let ResolutionFolder = __SOURCE_DIRECTORY__
+
+type githubEventType =
+    JsonProvider<"githubEventType.json", ResolutionFolder=ResolutionFolder>
```

Comment:
@realmarv resolutionFolder? Zahra, I told you twice already to use the same technique that we use in FiatEstimation.fs; we DO NOT use ResolutionFolder in FiatEstimation.fs

Reply:
I used both techniques used in src/GWallet.Backend/FiatValueEstimation.fs in geewallet and you say it's wrong. Where is FiatEstimation.fs? I only see FiatValueEstimation.fs in geewallet. And without ResolutionFolder JsonProvider can't find "githubEventType.json" file. It searches in "/tmp/nuget" folder for "githubEventType.json" file. If I don't use "githubEventType.json" file and copy its content in "detectNotUsingGitPush1by1.fsx" the script will become too long. What do you expect me to do?

Reply:
> Where is FiatEstimation.fs? 

I meant FiatValueEstimation.fs sorry.

> And without ResolutionFolder JsonProvider can't find "githubEventType.json" file

FiatValueEstimation.fs doesn't use any external file, therefore it doesn't need ResolutionFolder. I don't understand why this is so hard to grasp TBH.

> ... copy its content in "detectNotUsingGitPush1by1.fsx" the script will become too long

Who is charging you money for the length of scripts or commit messages? I told you many times that there's no issue with that, let alone this subjective matter dictating your mind to try other techniques (ResolutionFolder) that are causing you problems that are making this task much much much longer than it should have been.